### PR TITLE
docs: Add otlploghttp package in exporters README.md

### PR DESCRIPTION
**Pull request by arukiidou**
Add logs package in README.md

### suggested labels

- https://github.com/open-telemetry/opentelemetry-go/labels/area%3Alogs
- https://github.com/open-telemetry/opentelemetry-go/labels/documentation
- https://github.com/open-telemetry/opentelemetry-go/labels/Skip%20Changelog

### CODE REVIEW

**Comment by dmathieu** [comment]
How about sorting the packages (and signal columns) alphabetically?

File: exporters/README.md [hunk 1]
```diff
@@ -7,16 +7,19 @@ This package contains exporters for this purpose.
...
-|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc](./otlp/otlptrace/otlptracegrpc)   |         |   ✓    |
-|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp](./otlp/otlptrace/otlptracehttp)   |         |   ✓    |
-|                     [go.opentelemetry.io/otel/exporters/prometheus](./prometheus)                     |    ✓    |        |
-|            [go.opentelemetry.io/otel/exporters/stdout/stdoutmetric](./stdout/stdoutmetric)            |    ✓    |        |
-|             [go.opentelemetry.io/otel/exporters/stdout/stdouttrace](./stdout/stdouttrace)             |         |   ✓    |
-|                         [go.opentelemetry.io/otel/exporters/zipkin](./zipkin)                         |         |   ✓    |
+|                                           Exporter Package                                            | Logs | Metrics | Traces |
+|:-----------------------------------------------------------------------------------------------------:|:------:|:------:|:------:|
+|       [go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc](./otlp/otlplog/otlploggrpc)       |   ✓   |        |        |
```
Comment (dmathieu) [inline exporters/README.md:12]:
```suggestion
```

This package is still a skeleton, it's not implemented yet.

**Comment by arukiidou** [comment]
> How about sorting the packages (and signal columns) alphabetically?

I think that these sorted alphabetically by packages fullpath.
Sorry if I have missed something.

**Review by dashpole — APPROVED** [review]

**Review by dmathieu — APPROVED** [review]

File: exporters/README.md [hunk 2]
```diff
@@ -7,16 +7,18 @@ This package contains exporters for this purpose.
...
+| [go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc](./otlp/otlpmetric/otlpmetricgrpc) |        |   ✓   |        |
+| [go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp](./otlp/otlpmetric/otlpmetrichttp) |        |   ✓   |        |
+|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc](./otlp/otlptrace/otlptracegrpc)   |        |        |   ✓   |
+|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp](./otlp/otlptrace/otlptracehttp)   |        |        |   ✓   |
+|                     [go.opentelemetry.io/otel/exporters/prometheus](./prometheus)                     |        |   ✓   |        |
+|                [go.opentelemetry.io/otel/exporters/stdout/stdoutlog](./stdout/stdoutlog)              |   ✓   |        |        |
+|            [go.opentelemetry.io/otel/exporters/stdout/stdoutmetric](./stdout/stdoutmetric)            |        |   ✓   |        |
+|             [go.opentelemetry.io/otel/exporters/stdout/stdouttrace](./stdout/stdouttrace)             |        |        |   ✓   |
+|                         [go.opentelemetry.io/otel/exporters/zipkin](./zipkin)                         |        |        |   ✓   |
```
Comment (MrAlias) [inline exporters/README.md:21]:
```suggestion
|:-----------------------------------------------------------------------------------------------------:|:----:|:-------:|:------:|
|       [go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploghttp](./otlp/otlplog/otlploghttp)       |   ✓  |         |        |
| [go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc](./otlp/otlpmetric/otlpmetricgrpc) |      |   ✓     |        |
| [go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp](./otlp/otlpmetric/otlpmetrichttp) |      |   ✓     |        |
|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc](./otlp/otlptrace/otlptracegrpc)   |      |         |   ✓    |
|   [go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp](./otlp/otlptrace/otlptracehttp)   |      |         |   ✓    |
|                     [go.opentelemetry.io/otel/exporters/prometheus](./prometheus)                     |      |   ✓     |        |
|                [go.opentelemetry.io/otel/exporters/stdout/stdoutlog](./stdout/stdoutlog)              |   ✓  |         |        |
|            [go.opentelemetry.io/otel/exporters/stdout/stdoutmetric](./stdout/stdoutmetric)            |      |   ✓     |        |
|             [go.opentelemetry.io/otel/exporters/stdout/stdouttrace](./stdout/stdouttrace)             |      |         |   ✓    |
|                         [go.opentelemetry.io/otel/exporters/zipkin](./zipkin)                         |      |         |   ✓    |
```

**Review by MrAlias — APPROVED** [review]

**Pull request merged**